api/v1alpha2/template.go

```diff
@@ -49,11 +49,9 @@ type Action struct {
 	// +optional
 	Volumes []Volume `json:"volumes,omitempty"`
 
-	// NetworkNamespace defines the network namespace to run the container in. This enables access
-	// to the host network namespace.
-	// See https://man7.org/linux/man-pages/man7/namespaces.7.html.
+	// Namespaces defines the Linux namespaces this container should execute in.
 	// +optional
-	NetworkNamespace *string `json:"networkNamespace,omitempty"`
+	Namespaces *Namespaces `json:"namespaces,omitempty"`
```

Comment:
Shouldn't it be called `Namespace` instead of `Namespaces`? It has a struct type and within the struct there are no arrays/list defined.

Reply:
I pluralized it because there are multiple namespaces in the Namespaces data structure.

You can specify multiple namespaces simultaneously.

Reply:
Sure, but in this context, you only have can one namespace per Action, or am I missing something here? I mean I found it a bit confusing, especially if you are looking at it from a pure data structural point of view ;)

Reply:
I see what you mean. I'll update.